src/commands/force/user/list.ts

```diff
@@ -47,7 +47,7 @@ export class UserListCommand extends SfdxCommand {
       // if they passed in a alias and it maps to something we have an Alias.
       const alias = aliases.getKeysByValue(authData.getUsername())[0];
       const configAgg = ConfigAggregator.getInstance();
-      const defaultUserOrAlias = configAgg.getLocalConfig().get('defaultusername');
+      const defaultUserOrAlias = configAgg.getLocalConfig()?.get('defaultusername');
```

Comment:
Why only the localConfig? Why not `configAgg.getValue('defaultusername').value`?

Reply:
I think that would be an improvement.  I can't think of a reason not to aggregate it.

Reply:
did you mean `configAgg.getConfig()`
or should we use the non-instance method like
`ConfigAggregator.getValue('defaultusername')`

Reply:
Ohh woops, I meant `configAgg.getPropertyValue('defaultUsername')` but the static would work too.

Reply:
moot given the above.